scripts/setup/debians/ubuntu20_patches/boringssl/changelog

```diff
@@ -0,0 +1,27 @@
+android-platform-external-boringssl (8.1.0+r23-2ubuntu1+focal1) focal; urgency=medium
```

Comment:
To be honest, I like them there since they fit nicely with the other Debian builds. I couldn't find a place where they would fit that well on astrobee_platorm. They are actually more related to our Debian repo.

Reply:
I think it's great to have as much of the software as possible available for external developers to see and don't see much downside to keeping it in `astrobee` vs. hidden in `astrobee_platform`.

However (I didn't check this), are we making it easy for people to build only the necessary packages? Whatever standard installation instructions we give to external users should probably not build the stuff @bcoltin is talking about.